src/main/java/com/olgaboiar/mint/Header.java

```diff
@@ -30,25 +31,58 @@ String getDate() {
         return date;
     }
 
-    String prepareHeaders() {
-        if (allowedMethods != null) {
-            String methods = String.join(",", allowedMethods);
-            String accessControl = "Allow: " + methods;
-            headersToSend = statusLine + "\n"
-                    + contentType + "\n"
-                    + date + "\n"
-                    + accessControl + "\n"
-                    + blankLine;
-        } else {
-            headersToSend = statusLine + "\n"
-                + contentType + "\n"
-                + date + "\n"
-                + blankLine;
+    public ArrayList<String> getHeaders() {
+        ArrayList<String> headers = new ArrayList<String>();
+        headers.add(getStatusLine());
+        headers.add(getContentType());
+        headers.add(getDate());
+        if (allowedMethodsExist()) {
+            String aloowHeader = createAllowHeader();
+            headers.add(aloowHeader);
```

Comment:
`aloowHeaer` 👉 `allowHeader`

Reply:
fixed typo in https://github.com/olgaboiar/MintHTTPServer/pull/10/commits/2d3c8d12991bbb9e025749db2db0799324b6d691